alternator/executor.cc

```diff
@@ -4120,11 +4162,12 @@ future<executor::request_return_type> executor::get_item(client_state& client_st
     co_await verify_permission(_enforce_authorization, client_state, schema, auth::permission::SELECT);
     co_return co_await _proxy.query(schema, std::move(command), std::move(partition_ranges), cl,
             service::storage_proxy::coordinator_query_options(executor::default_timeout(), std::move(permit), client_state, trace_state)).then(
-            [this, schema, partition_slice = std::move(partition_slice), selection = std::move(selection), attrs_to_get = std::move(attrs_to_get), start_time = std::move(start_time), add_capacity=std::move(add_capacity)] (service::storage_proxy::coordinator_query_result qr) mutable {
+            [stats, this, schema, partition_slice = std::move(partition_slice), selection = std::move(selection), attrs_to_get = std::move(attrs_to_get), start_time = std::move(start_time), add_capacity=std::move(add_capacity)] (service::storage_proxy::coordinator_query_result qr) mutable {
 
+        stats->api_operations.get_item_latency.mark(std::chrono::steady_clock::now() - start_time);
         _stats.api_operations.get_item_latency.mark(std::chrono::steady_clock::now() - start_time);
 
-        return make_ready_future<executor::request_return_type>(make_jsonable(describe_item(schema, partition_slice, *selection, *qr.query_result, std::move(attrs_to_get), add_capacity, _stats.rcu_half_units_total)));
+        return make_ready_future<executor::request_return_type>(make_jsonable(describe_item(schema, partition_slice, *selection, *qr.query_result, std::move(attrs_to_get), add_capacity, stats->rcu_half_units_total)));
```

Comment:
I'm probably missing something here, but don't you need to increment both _stats and the per-table stats - and not just per-table stats?

Reply:
This part is just for the rcu